core/IssueQueue.cpp

```diff
@@ -0,0 +1,228 @@
+// <IssueQueue.cpp> -*- C++ -*-
+
+#include "IssueQueue.hpp"
+#include "CoreUtils.hpp"
+namespace olympia {
+const char IssueQueue::name[] = "issue_queue";
+
+IssueQueue::IssueQueue(sparta::TreeNode *node, const IssueQueueParameterSet *p)
+    : sparta::Unit(node),
+      reg_file_(olympia::coreutils::determineRegisterFile(
+          node->getGroup().substr(node->getGroup().find("_") + 1))),
+      scheduler_size_(p->scheduler_size), in_order_issue_(p->in_order_issue) {
```

Comment:
Please put the `{` on the next line

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)